py-core/app/util/docker-compose/reference-snapshot/docker-compose.yml

```diff
@@ -0,0 +1,88 @@
+version: '3'
+
+services:
+
+  tezos-node:
+    image: tezos/tezos:latest-release
+    restart: always
+    ports:
+      - "8732:8732"
+    expose:
+      - 8732
+    command: "NODE START COMMAND"
+    networks:
+      - tezos
+    volumes:
+      - ./tezos-data:/var/run/tezos
+    logging:
+      driver: "json-file"
+
+  conseil-postgres:
+    image: postgres:11.6
+    restart: always
+    environment:
+      POSTRGES_PASSWORD: "password"
+      POSTGRES_USER: "conseil"
```

Comment:
Things like passwords should be stored in a separate config file and included dynamically.

Reply:
These are passwords for internal docker containers running locally, basically the point is that there isn't any config required, the thing works off the bat

Reply:
By exposing the port, you imply that a user should be able to connect to the db outside of docker if they wanted to. Say I wanted to inspect the db with pgadmin, I should be able input/change the default credentials/port from a centralized location rather finding it in the docker compose. (port 5432, for example, might be taken already)

Reply:
Yes, I see your point, since you don't actually publish the port I think this is fine. Something to think about in the future should you end up exposing some of these ports though